JHUGenMELA/MELA/src/Mela.cc

```diff
@@ -791,16 +793,23 @@ void Mela::computeVHAngles(
       &(mothers.at(1).second), mothers.at(1).first
     );
 
+    const auto pV = aparts[0].second + aparts[1].second;
```

Comment:
1) Prefer to use directly
mV = (aparts[0].second + aparts[1].second).M();
mVstar = (pV + daughters[0].second + daughters[1].second + daughters[2].second + daughters[3].second).M();
instead of storing temporary variables through the end of the function that are not used anywhere else.
2) Migrate this implementation in TUtil instead and call from there.

Reply:
1. this is slightly more efficient because it only adds the jet momenta once
2. I was being consistent with computeDecayAngles.

I don't think either of these really matter.  If you do feel free to change them.